tools/aie2xclbin/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 configure_file(configure.h.in configure.h)
 target_include_directories(aie2xclbin PRIVATE "${CMAKE_CURRENT_BINARY_DIR}")
 
+find_library (UUID uuid)
```

Comment:
Maybe add REQUIRED - you're failing in windows because it's missing

Reply:
yeah, I'm trying to figure out a windows build so I can fix it there.